android-stellar-sdk/src/main/java/org/stellar/sdk/responses/Response.java

```diff
@@ -7,14 +7,18 @@
 
   public void setHeaders(String limit, String remaining, String reset) {
     try {
-      this.rateLimitLimit = Integer.parseInt(limit);
-      this.rateLimitRemaining = Integer.parseInt(remaining);
-      this.rateLimitReset = Integer.parseInt(reset);
+      this.rateLimitLimit = safeParse(limit);
```

Comment:
check who calls setHeaders and if it is ok not to throw an exception when number is null or empty